Makefile

```
@@ -65,6 +72,23 @@ compile-docker:	 	## Bundles Infection into a PHAR using docker
+	IMAGE_NAME=$(SBX_IMAGE_NAME) \
+	IMAGE_TAG=$(SBX_IMAGE_TAG) \
+	IMAGE_REF=$(SBX_IMAGE_REF) \
+	IMAGE_PLATFORM=$(SBX_IMAGE_PLATFORM) \
+	IMAGE_CACHE_FROM=$(SBX_IMAGE_CACHE_FROM) \
+	IMAGE_CACHE_TO=$(SBX_IMAGE_CACHE_TO) \
+	PHP_VERSION=$(SBX_PHP_VERSION) \
+	./devTools/sbx/build-image.sh
+
+.PHONY: sbx-image-smoke-test
+sbx-image-smoke-test:	## Verifies the PHP sbx image contains the expected tooling
+sbx-image-smoke-test:
```

checkmake (0.3.2)

[warning] Target "sbx-image-smoke-test" defined multiple times (lines 88 and 89).

(uniquetargets)